service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderGenerationService.java

```diff
@@ -132,7 +132,7 @@ private String getScheduleProvided(CaseManagementOrder caseManagementOrder) {
 
     private DocmosisJudgeAndLegalAdvisor getJudgeAndLegalAdvisorData(JudgeAndLegalAdvisor judgeAndLegalAdvisor) {
         return DocmosisJudgeAndLegalAdvisor.builder()
-            .judgeTitleAndName(defaultIfBlank(formatJudgeTitleAndName(judgeAndLegalAdvisor), DEFAULT))
+            .judgeTitleAndName(formatJudgeTitleAndName(judgeAndLegalAdvisor))
```

Comment:
Do we still not want to display the default string in the draft?

Reply:
formatJudgeTitleAndName method handles

Reply:
`  public static String formatJudgeTitleAndName(JudgeAndLegalAdvisor judgeAndLegalAdvisor) {
        return Optional.ofNullable(judgeAndLegalAdvisor)
            .filter(judge -> judge.getJudgeTitle() != null)
            .map(JudgeAndLegalAdvisorHelper::mapJudgeOrAdvisor)
            .orElse("");
    }`

Reply:
Somehow in my head had the 2 constants the wrong way round, thinking it would be the `HEARING_EMPTY_PLACEHOLDER`. Need more sleep